aws/resource_aws_elastic_beanstalk_environment.go

```diff
@@ -970,9 +970,11 @@ func (e beanstalkEnvironmentError) Error() string {
 
 type beanstalkEnvironmentErrors []*beanstalkEnvironmentError
 
-func (e beanstalkEnvironmentErrors) Len() int           { return len(e) }
-func (e beanstalkEnvironmentErrors) Swap(i, j int)      { e[i], e[j] = e[j], e[i] }
-func (e beanstalkEnvironmentErrors) Less(i, j int) bool { return e[i].eventDate.Before(*e[j].eventDate) }
+func (e beanstalkEnvironmentErrors) Len() int      { return len(e) }
+func (e beanstalkEnvironmentErrors) Swap(i, j int) { e[i], e[j] = e[j], e[i] }
+func (e beanstalkEnvironmentErrors) Less(i, j int) bool {
+	return e[i].eventDate.Before(*e[j].eventDate)
```

Comment:
This is a super minor nit-pick, but it would have been nice to have a separate commit for a seemingly unrelated reformat.

Reply:
yeah, this changed was introduced with `make fmt`, sorry for not splitting it up in diff commits. If you want, I could give it a crack and split the commit up. Hopefully it won't affect this PR ?

Reply:
No need to stress it on my part, I just noticed and thought to share a nitpick 🙂As far as affecting - I hope not! I'm just an interested user of this functionality in this case with no official reviewing rights, so I hope this doesn't affect anything.